kernel/safe_typing.mli

```diff
@@ -169,6 +169,9 @@ val add_module_parameter :
   MBId.t -> Entries.module_struct_entry -> Declarations.inline ->
     Mod_subst.delta_resolver safe_transformer
 
+(** returns the number of module parameters *)
+val module_num_parameters : safe_environment -> int
```

Comment:
Don't you want to also return whether we are in a module type or a module?

Reply:
done